index.html

```diff
@@ -5,12 +5,11 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
```

Comment:
- [ ] [OPTIONAL] Great job adding the `h1` tag to the body 👏  👍 , but you don't need to delete this line.
This is the link to the external style sheet `style.css` we need it to have the color of `Hello Microverse!` green like before.
Please add it back.